test/grpc/unary.cc

```diff
@@ -93,5 +90,26 @@ TEST(UnaryTest, Success) {
   server->Wait();
 }
 
+TEST(UnaryTest, SuccessWithDefaultChannel) {
+  TestUnaryWithClient([](Borrowable<CompletionPool>&& pool, int port) {
```

Comment:
optional: `port` can be const, here and below

Reply:
👍🏻